test/Connection.js

```diff
@@ -211,8 +211,8 @@ describe('Connection', () => {
     behavesLikeRequestWithBody('post', requestPath, 'application/octet-stream', sendRequest)
     behavesLikeTimeoutableRequest('post', requestPath, sendRequest)
     behavesLikeRequestWithAPIKey(requestPath, sendRequest)
-    behavesLikeRequestWithProgress('Download', 'post', requestPath, sendRequest)
-    behavesLikeRequestWithProgress('Upload', 'post', requestPath, sendRequest)
+    // behavesLikeRequestWithProgress('Download', 'post', requestPath, sendRequest)
```

Comment:
Ah, are these cases failed because of outdated `xhr-mock`?
Commenting out is OK but could you please add some TODO comment?

Reply:
Yes, I forgot to uncomment them before pushing. I'll add a TODO comment and I'll toy around with fixing them later.

Reply:
Thank you! And the fix is so helpful for us :)